streams.yaml

```diff
@@ -14,57 +14,132 @@ proxima.polygon-mumbai.blocks.1_0:
   description: "block headers of polygon mumbai testnet"
   labels:
     encoding: "json"
+proxima.eth-main.blocks-sync.1_0:
+  description: "block ids of ethereum mainnet"
+  labels:
+    encoding: "proto"
+proxima.eth-goerli.blocks-sync.1_0:
+  description: "block ids of ethereum goerli testnet"
+  labels:
+    encoding: "proto"
+proxima.polygon-main.blocks-sync.1_0:
+  description: "block ids of polygon mainnet"
+  labels:
+    encoding: "proto"
+proxima.polygon-mumbai.blocks-sync.1_0:
+  description: "block ids of polygon mumbai testnet"
+  labels:
+    encoding: "proto"
+
 proxima.erc20.eth-main.events.1_0:
   description: "all erc20 events on eth-main"
   labels:
     encoding: "json"
+    legacy: true
 proxima.erc20.eth-goerli.events.1_0:
   description: "all erc20 events on eth-goerli"
   labels:
     encoding: "json"
+    legacy: true
 proxima.erc20.polygon-main.events.1_0:
   description: "all erc20 events on polygon-main"
   labels:
     encoding: "json"
+    legacy: true
 proxima.erc20.polygon-mumbai.events.1_0:
   description: "all erc20 events on polygon-mumbai"
   labels:
     encoding: "json"
+    legacy: true
+proxima.erc20.eth-main.events.2_0:
+  description: "all erc20 events on eth-main"
+  labels:
+    encoding: "json"
```

Comment:
if it's a new stream, it's proto encoded

Reply:
Thank you, missed that